### PR TITLE
CN2: Remove time consuming tests and prints

### PR DESCRIPTION
**Pull request by janezd**
Tests for pickling and handling missing values doubled the execution time of the entire suite. I tried and failed to tune the parameters to decrease the time to something reasonable.

### CODE REVIEW

File: Orange/tests/test_classification.py [hunk 1]
```diff
@@ -285,6 +289,8 @@ def test_all_models_work_after_unpickling(self):
                 print('%s cannot be used with default parameters' % learner.__name__)
                 traceback.print_exc()
                 continue
+            if isinstance(learner, _RuleLearner):
+                continue
```
Comment (kernc) [inline Orange/tests/test_classification.py]: Missing comment explaining why.

File: Orange/tests/test_classification.py [hunk 2]
```diff
@@ -240,6 +242,8 @@ def test_missing_class(self):
                 learner = learner()
                 if isinstance(learner, NuSVMLearner):
                     learner.params["nu"] = 0.01
+                if isinstance(learner, _RuleLearner):
+                    continue
```
Comment (kernc) [inline Orange/tests/test_classification.py]: Missing comment explaining why.

File: Orange/classification/rules.py [hunk 3]
```diff
@@ -4,6 +4,9 @@
 from collections import namedtuple
 import bottleneck as bn
 import numpy as np
+import time
+
+import pickle
```
Comment (kernc) [inline Orange/classification/rules.py]: Forgot this here?

Comment (janezd) [inline Orange/classification/rules.py]: Yes, thanks. All fixed.

**Pull request merged**